src/plugins/intel_cpu/src/ngraph_transformations/snippets_mark_skipped.cpp

```diff
@@ -458,6 +484,9 @@ bool SnippetsMarkSkipped::run_on_model(const std::shared_ptr<ov::Model> &m) {
         } else if (isSuitableSubtractAsZeroPointsParent(node)) {
             SetSnippetsNodeType(node, snippets::pass::SnippetsNodeType::SkippedByPlugin);
             channelAxis = DEFAULT_AXIS;
+        } else if (enableBF16 && isSuitableConvert(node)) {
```

Comment:
@dmitry-gorokhov The skipped logic requires that CPU plugin decides to use BF16 otherwise this convert may not be correct since snippets may construct it own convert subgraph which has similar pattern but has different meaning e.g.  example code [here](https://github.com/openvinotoolkit/openvino/blob/master/src/tests/functional/plugin/shared/include/snippets/convert.hpp)

Reply:
As we agreed lets provide comment why the skip is needed and create feature request on Snippets